### PR TITLE
2) Add and Configure the Main Menu

### PR DESCRIPTION
**Pull request by crudobaker**
This PR removes the `/hola` command, which was the one used to display the main functions, and replaces it for the use of the **Main Menu** button.

The Main Menu options are configured in the `configureCommands` middleware. It uses the user form the `ctx.state` and depending on the role of the user, it adds the commands related to the role.

image

_(this is the Coordinator View, which includes the Admin Guards option)_

It also splits the bot configuration into `commands` and `actions` sections. Both directories have a file per command/action, and one `index.js` file which contains the configuration function.

### CODE REVIEW

File: bot/src/telegram/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { GET_LOADED_GUARDS } from "bot/src/telegram/actions/get-loaded-guards.js";
+import { GET_NOT_COVERED_GUARDS } from "bot/src/telegram/actions/get-not-covered-guards.js";
+import { GET_GUARD_INFORMATION } from "bot/src/telegram/actions/get-guard-information.js";
+import { DELETE_GUARD } from "bot/src/telegram/actions/delete-guard.js";
+import { GET_AVAILABLE_USER_FOR_ASSIGN } from "bot/src/telegram/actions/get-available-user-for-assign.js";
+import { ASSIGN_GUARD } from "bot/src/telegram/actions/assign-guard.js";
+
+export default function configureActions(bot) {
+  bot.action(GET_LOADED_GUARDS.name, GET_LOADED_GUARDS.apply);
```
Comment (pablotocalini) [inline bot/src/telegram/actions/index.js:9]: porque apply?

File: bot/src/telegram/index.js [hunk 2]
```diff
@@ -1,261 +1,24 @@
...
-  }
+  ctx.reply(
+    `Hola ${user.firstName} 👋. Soy El Guardian 🤖.\n\n Presiona el botón menú para ver las opciones disponibles.`
+  );
 });
 
 bot.launch();
 
 // Enable graceful stop
 process.once("SIGINT", () => bot.stop("SIGINT"));
```
Comment (pablotocalini) [inline bot/src/telegram/index.js:23]: what???????

File: bot/src/telegram/actions/index.js (hunk 1 above)
Comment (crudobaker) [reply bot/src/telegram/actions/index.js:9]: Es como decir "aplicar" la función, la acción. Igual luego lo cambie por execute, en el otro PR.

File: bot/src/telegram/index.js (hunk 2 above)
Comment (crudobaker) [reply bot/src/telegram/index.js:23]: Jaja esas dos configuraciones les ví en varios ejemplos. Básicamente lo que hace es que al finalizar el proceso de node, le dice al bot que finalice.

**Review by pablotocalini — APPROVED** [review]

**Pull request merged**